### PR TITLE
Fix build path and graph dep issue

### PR DESCRIPTION
**Pull request by pddpd**
ATT

### CODE REVIEW

File: packages/ava/tsconfig.json [hunk 1]
```diff
@@ -5,5 +5,5 @@
     "declaration": true,
     "declarationDir": "./lib"
   },
-  "include": ["__tests__"]
```
Comment (pddpd) [inline packages/ava/tsconfig.json:8]: it causes build path issue

File: packages/ava/package.json [hunk 2]
```diff
@@ -64,6 +64,7 @@
     "tslib": "^2.3.1"
   },
   "devDependencies": {
+    "@antv/algorithm": "^0.1.25",
```
Comment (Esorakouki) [inline packages/ava/package.json:67]: why is dev?

Comment (pddpd) [reply packages/ava/package.json:67]: graph deps it, but now we just move graph files, and have not finished it

I've @chenluli here
https://github.com/antvis/AVA/pull/505/files#diff-b631b9f3e6d513f3d244c081cbef910e7a7afa54c2c782ea607d48a6cbd76234R1

**Review by Esorakouki — APPROVED** [review]
LGTM

**Pull request merged**